versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -256,6 +256,23 @@ your website.
 
 :::
 
+### Performance: remove chunks preloading
+
+We've
+[removed the chunks preloading feature](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2120)
+as it led to blocking network requests that could reduce the
+
+<abbr title="Largest Contentful Paint">LCP</abbr> /<abbr title="First Contentful Paint">
+  FCP
+</abbr> of your app. From our analysis it improves the performance of the
+application and is a good default.
```

Comment:
The <abbr> tag is being added to a new line by prettier, this is a pitfall in mdx, where the linter does not know if it should indent markdown or jsx, to avoid this, I suggest converting the link to html and wrapping it in a fragment

```suggestion
<>
  We've{" "}
  <a href="https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2120">
    removed the chunks preloading feature
  </a>{" "}
  as it led to blocking network requests that could reduce the{" "}
  <abbr title="Largest Contentful Paint">LCP</abbr> /{" "}
  <abbr title="First Contentful Paint"> FCP</abbr> of your app. From our
  analysis it improves the performance of the application and is a good default.
</>
```

| Before | After |
| ------- | ------ |
| ![image](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/e1106e0b-ef49-467d-8696-7b58cfa7257b) | ![image](https://github.com/front-commerce/developers.front-commerce.com/assets/39598117/8ce934e7-d8c6-46c7-a255-72b71e23fa91) |

Reply:
Thanks for investigating. I've done this.